crates/miden-objects/src/account/storage/mod.rs

```diff
@@ -271,9 +271,9 @@ fn slots_as_elements(slots: &[StorageSlot]) -> Vec<Felt> {
 }
 
 /// Computes the commitment to the given slots
-fn build_slots_commitment(slots: &[StorageSlot]) -> Digest {
+pub(crate) fn build_slots_commitment(slots: &[StorageSlot]) -> Digest {
```

Comment:
Why does this need to be `pub(carte)`? - it doesn't seem to be used outside of this module

Reply:
I thought I could use it at first and did not commit the rollback of the change later, removed the `(crate)`